pkgs/applications/misc/slic3r/prusa3d.nix

```diff
@@ -33,21 +33,25 @@ let
 in
 stdenv.mkDerivation rec {
   name = "slic3r-prusa-edition-${version}";
-  version = "1.40.1";
+  version = "1.41.0";
 
   enableParallelBuilding = true;
 
   buildInputs = [
     cmake
     curl
-    perl
-    makeWrapper
     eigen
     glew
+    gtest
+    makeWrapper
```

Comment:
This should be in `nativeBuildInputs` since it is only used at build time, never at runtime.

Reply:
I have moved them, as well as cmake